libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -39,9 +40,15 @@ class AP_Airspeed
     
     // constructor
     AP_Airspeed();
+    AP_Airspeed(AP_AHRS &ahrs);
 
     void init(void);
 
+    enum blend_type {
+        BLEND_NONE=0,
+        BLEND_FULL=1,
```

Comment:
if it's only two states I suggest a boolean enable

Reply:
I consider more options in future. Anyway in parameters it will take the same amount of mem as far as I understand.